chains/atomic/atomicmock/shared_memory.go

```diff
@@ -42,10 +43,10 @@ func (m *SharedMemory) EXPECT() *SharedMemoryMockRecorder {
 }
 
 // Apply mocks base method.
-func (m *SharedMemory) Apply(arg0 map[ids.ID]*atomic.Requests, arg1 ...database.Batch) error {
+func (m *SharedMemory) Apply(requests map[ids.ID]*atomic.Requests, batches ...database.Batch) error {
```

Comment:
Note mockgen v0.5 does set the argument names correctly (nice to have!), but that also introduces most of the diffs in this PR.